beancount/parser/printer_test.py

```diff
@@ -230,7 +232,7 @@ def test_Transaction(self, entries, errors, __):
           Assets:Account1         1 USD @ 0 OTHER
           Assets:Account2         1 CAD @ 0 OTHER
 
-        2014-06-10 * "Entry with escaped \\"symbols\\" \\ \\r \\n"
+        2014-06-10 * "Entry with escaped \"symbols\" \ \r \n"
```

Comment:
Why this change?

Reply:
This is marked as resolved, but the change is still there.

Reply:
for better readability

Reply:
Please drop this change. Readability is subjective. Especially in this case.: for me, having to scan several lines up to realize that this is a raw string and that the back-slash escaping is not interpreted by Python is much less readable than the double escapes.